packages/sdk/src/libs/clients/emp/client.ts

```diff
@@ -16,13 +16,13 @@ export interface EventState {
   expired?: boolean;
 }
 
-export type RequestTransferPositionExecuted = GetEventType<ExpiringMultiParty, "RequestTransferPositionExecuted">;
-export type PositionCreated = GetEventType<ExpiringMultiParty, "PositionCreated">;
-export type NewSponsor = GetEventType<ExpiringMultiParty, "NewSponsor">;
-export type SettleExpiredPosition = GetEventType<ExpiringMultiParty, "SettleExpiredPosition">;
-export type Redeem = GetEventType<ExpiringMultiParty, "Redeem">;
-export type Withdrawal = GetEventType<ExpiringMultiParty, "Withdrawal">;
-export type LiquidationCreated = GetEventType<ExpiringMultiParty, "LiquidationCreated">;
+export type RequestTransferPositionExecuted = GetEventType<Instance, "RequestTransferPositionExecuted">;
+export type PositionCreated = GetEventType<Instance, "PositionCreated">;
+export type NewSponsor = GetEventType<Instance, "NewSponsor">;
+export type SettleExpiredPosition = GetEventType<Instance, "SettleExpiredPosition">;
+export type Redeem = GetEventType<Instance, "Redeem">;
+export type Withdrawal = GetEventType<Instance, "Withdrawal">;
+export type LiquidationCreated = GetEventType<Instance, "LiquidationCreated">;
```

Comment:
you can ignore this, just renaming vars to make it easier to make templates from these files